stdlib/jms/src/main/ballerina/jms/message.bal

```diff
@@ -19,158 +19,169 @@
 # application data being sent.
-public type Message object {
+public type Message client object {
+    private Session session;
+    private MessageType msgType;
+
+    # The `Message` object is initialized with a `Session` object and `MessageType` indicating the type of the message.
+    # 
+    # + session - The session from which the message would be created
+    # + msgType - The type of the message
+    # + return - `error` if `Message` creation fails
+    public function __init(Session session, MessageType msgType) returns error? {
+        self.session = session;
+        self.msgType = msgType;
+        return self.createMessage();
+    }
+
+    function createMessage() returns error? = external;
+
+    # Sets the payload/body of the message. 
+    # If the `MessageType` is `TEXT_MESSAGE` an attempt would be made to convert the data to string type and the 
+    # body/content of the `Message` would be replaced.
+    # If the `MessageType` is `MAP_MESSAGE` then only `map` type content is allowed and the data would be replaced. 
+    # Otherwise an error is returned.
+    # If the `MessageType` is `BYTES_MESSAGE` or `STREAM_MESSAGE` this method can be called multiple times to populate 
+    # the message without replacing the content. If the content is of types `map`, `xml` or `json`, they will be 
+    # converted to string. 
+    # The data types `json`, `xml` and `map` are converted to `string` before sending except for `MAP_MESSAGE` type 
+    # which functions as described above.
+    # 
+    # + content - the content of the message.
+    # + return - If an error occurred while setting the payload or on content mismatch.
+    public function setPayload(@untainted MessageContent content) returns error? = external;
+
+    # Returns the payload of the message.
+    # If the `MessageType` is  `TEXT_MESSAGE` returns string data or attempts to convert the message to the given 
+    # optional `DataType`. The `bytesLength` parameter if set would be ignored.
+    # If the `MessageType` is `MAP_MESSAGE` then a `map` is returned regardless of the `DataType` and `bytesLength`
+    # If the `MessageType` is `BYTES_MESSAGE` or `STREAM_MESSAGE` the `DataType` has to be mentioned as it cannot 
+    # be deduced. This method can be called multiple times to receive the available data. If the `DataType` is `BYTES` 
+    # then the `bytesLength` has to be specified, otherwise the number of bytes to be read cannot be deduced. 
+    #
+    # + dataType - this is optional for `TEXT_MESSAGE` and `MAP_MESSAGE` types and mandatory for `BYTES_MESSAGE` and
+    # `STREAM_MESSAGE` types to deduce the data to be returned.
+    # + bytesLength - this is optional and is ignored for other dataTypes except for `BYTES` dataType in 
+    # `BYTES_MESSAGE` and `STREAM_MESSAGE`
+    # + return - If an error occurred retrieving the payload from the `Message` or if the required parameters are not
+    # set
```

Comment:
```suggestion
   
```
We might not need a new line only for one word.

Reply:
We need to break line at 120 characters